src/Services/NodePastDealsService.ts

```diff
@@ -28,4 +29,22 @@ export class NodePastDealsService {
         }
         return await NodePastDealModel.create({cid, state, size, provider, price, duration, nodeId});
     }
+
+    public async fetchNodePastDeals(userId: number, from: number, to: number, orderBy: string) {
+        return await NodePastDealModel.findAll({
+            attributes: ['id', 'cid', 'state', 'size', 'provider',
```

Comment:
any reason why not all attributes but specifically hardcoded here?

Reply:
At some point I had to do this to retrieve only records from this table without the data from nodes table. Now with the include below this is no longer needed, my oversight :)